### DISCUSSION TITLE
Neutral color not exists by default !

### DISCUSSION THREAD

**Discussion by MuhammadMoftah** [tailwindlabs/tailwindcss#8303] [Help]
**What version of Tailwind CSS are you using?**
v3 

**What build tool (or framework if it abstracts the build tool) are you using?**
I'm using "@nuxtjs/tailwindcss": "^5.0.3"

**What version of Node.js are you using?**
16.14

**What browser are you using?**
Chrome, Firefox, Egde

**What operating system are you using?**
Windows

**Describe your issue**
At Tailwind V3 documentation their is color called (Neutral) by default but 
Neutral color not exists by default !

**Comment by adamwathan** [comment]
Definitely does exist: https://play.tailwindcss.com/vDiGALGCn5

Can you create a reproduction? Can't help without actually seeing how you have everything set up.

**Comment by shrekuu** [comment]
@MuhammadMoftah  You must have installed DaisyUI. 😅 You have to either modify daisyUI config or modify TaiwindCSS config. Here is some solution: https://github.com/saadeghi/daisyui/issues/683
